boards/arm/stm32h747i_disco/doc/index.rst

```diff
@@ -63,6 +63,10 @@ The current Zephyr stm32h747i_disco board configuration supports the following h
 +-----------+------------+-------------------------------------+
 | GPIO      | on-chip    | gpio                                |
 +-----------+------------+-------------------------------------+
+| ETHERNET  | on-chip    | ethernet                            |
++-----------+------------+-------------------------------------+
+| RNG       | on-chip    | True Random number generator        |
++-----------+------------+-------------------------------------+
 
```

Comment:
So yes, this board won't have ETH working until the user changes the SB8 and SB21 configuration. That's not ideal...
We have to maybe add the warning to the doc with an (*) ?

```suggestion
| ETHERNET  | on-chip    | ethernet  (*)                       |
+-----------+------------+-------------------------------------+
| RNG       | on-chip    | True Random number generator        |
+-----------+------------+-------------------------------------+

(*) From UM2411 Rev 4:
   With the default setting, the Ethernet feature is not working because of
   a conflict between ETH_MDC and SAI4_D1 of the MEMs digital microphone.
   Make sure you have SB8 closed and SB28 open to get Ethernet working.
```

Reply:
Thank you. I am absolutely agree - this belongs to be mentioned in documentation.